api/routes/auth.js

```diff
@@ -52,7 +54,7 @@ router.post('/login', controllers.login);
  *     summary: Register  a user
  *     consumes:
  *       - application/json
-  *     requestBody:
+ *     requestBody:
```

Comment:
Maybe it necessary to add to the description information about the header token.

Reply:
It's a big task, cause we need to pass the token in header in a lot of services. So we should rewrite a lot of logic, and after this, change swagger spec.

Reply:
Ok you are right)